paper.tex

```diff
@@ -1004,17 +1022,19 @@ \subsection*{Illustrative examples}
 
 \section*{Discussion}
 
-The key contribution of this paper is to rederive the classical KYF
-ARG likelihood function in a way that supports partial and imprecise
-input, using a novel backwards-time derivation of the SMC.
+The key contribution of this paper is to rederive a classical
+ARG likelihood function in a way that supports the sort of partial information
+provided by some modern ARG inference tools,
+using a novel backwards-time derivation of the SMC.
 The underlying process that we reason about probabilistically is still
-composed of events affecting lineages, but, acknowledging that we 
-do not necessarily observe or estimate these events, we calculate 
-the possible contributions of various type of event to the inheritance
-relationships that are provided as input.
-Although we can associate a likelihood with any ARG in \texttt{tskit}
-format, there are some caveats. By treating each edge independently we can
-naturally deal with the various degrees of completeness to which inheritance
+composed of events affecting lineages, but, because we
+do not necessarily observe or estimate these events,
+we integrate over the possible timings implied by the information
+in the input ARG.
```

Comment:
I think you were trying to make a point related to different types of ARG but I think that will go over the heads of the readership.